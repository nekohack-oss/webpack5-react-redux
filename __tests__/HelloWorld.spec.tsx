import * as React from 'react'
import renderer from 'react-test-renderer'
import HelloWorld from '../src/components/HelloWorld'

describe('Content', () => {
    it('render correctly', () => {
        const tree = renderer
            .create(<HelloWorld msg="Hello React 16.13" />)
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})
