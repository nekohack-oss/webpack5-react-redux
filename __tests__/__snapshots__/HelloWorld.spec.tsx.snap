// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content render correctly 1`] = `
Array [
  <h1>
    Hello React 16.13
  </h1>,
  <button
    onClick={[Function]}
  >
    count is: 
    0
  </button>,
  <p>
    Edit 
    <code>
      components/HelloWorld.tsx
    </code>
     to test hot module replacement.
  </p>,
]
`;
